src/WebJobs.Script.WebHost/DependencyInjection/DependencyValidator/DependencyValidator.cs

```diff
@@ -56,8 +56,7 @@ private static ExpectedDependencyBuilder CreateExpectedDependencies()
                 .Expect<AzureMonitorDiagnosticLoggerProvider>()
                 .Expect<FunctionFileLoggerProvider>()
                 .Expect<HostFileLoggerProvider>()
-                .Expect<SystemLoggerProvider>()
-                .Expect<UserLogMetricsLoggerProvider>();
```

Comment:
I don't see why we need to pay the price of this UserLogMetricsLogger just to count calls to user logs. Is anyone actually ever looking at that data?

Reply:
Shoebox needed us to count them. They wouldn't let us move to their infrastructure until we had some kind of load detail about what we'd be sending them. I really don't think it's required anymore since we're up-and-running on them now.